src/styles/main.scss

```
@@ -1,3 +1,45 @@
+$size: 300px;
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+}
+
 body {
   margin: 0;
 }
+
+.block {
+  display: flex;
```

```suggestion
  display: grid;
```
You could use display: grid for this purpose as well